common/src/main/java/net/dodogang/plume/mixin/client/CreativeInventoryScreenMixin.java

```diff
@@ -17,12 +15,13 @@
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 
+import java.util.ArrayList;
 import java.util.List;
 
 @Environment(EnvType.CLIENT)
 @Mixin(CreativeInventoryScreen.class)
 public abstract class CreativeInventoryScreenMixin extends AbstractInventoryScreen<CreativeInventoryScreen.CreativeScreenHandler> {
-    private final List<ItemGroupTabWidget> plume_tabButtons = Lists.newArrayList();
+    private final List<ItemGroupTabWidget> plume_tabWidgets = new ArrayList<>();
```

Comment:
Is there a reason this variable has `plume_` prepended to it? Is `tabWidgets` already a field in `AbstractInventoryScreen`?

Reply:
I've gotten into the habit of just appending the mod id to any non-shadowed methods or fields in mixins, which may help in crash reporting or mixin exports? I'm not sure as they haven't been necessarily useful to me yet, but yeah

Reply:
Ooh, I somehow didn't notice that was a mixin. I even thought, "hey, maybe that's a mixin", but somehow missed it. Makes sense